### ISSUE TITLE
Button to clean old photo backup

### ISSUE THREAD

**Issue by cansado2930** [nextcloud/android#9620]
Hello,
I would like to request an improvement, a button that the user can request to app to clean files from phone that there are just a copy on the cloud. For example, one case of use, The user has synced camera photos.. and he has to clean photos of phone in order to free space, so with this button, app deletes photos of phone that they have a copy on cloud. Where put this button? As app can sync a lot of folders.... on the menu of synchronization of each folder, the button can be put here, in order to free files of the folder.
Thank you

**Comment by AlvaroBrey** [comment]
Although it isn't the exact same functionality, there is an option in the folder sync configuration to delete files _when they are uploaded_. It doesn't work for files downloaded in the past though.

**Comment by cansado2930** [comment]
Yes, I know this option, but the app has synced more than 1000 photos of camera. If I change the mode, will app delete photos from phone that exist on cloud before?
